### ISSUE TITLE
Module broken with latest terraform release (v0.15.0)

### ISSUE THREAD

**Issue by deepankarm** [jina-ai/terraform-jina-jinad-aws#4]
`jina-ai/jinad-aws/jina` is broken with the [latest tf release](https://www.hashicorp.com/blog/announcing-hashicorp-terraform-0-15-general-availability).

> │ Error: Output refers to sensitive values
│ 
│   on .terraform/modules/jinad.keypair/outputs.tf line 9:
│    9: output "private_key_pem" {
│ 
│ Expressions used in outputs can only refer to sensitive values if the sensitive attribute is true.

**Comment by alanthssss** [comment]
I read the terraform v0.15.0  docs.
found these:
> Provider-defined sensitive attributes will now be redacted throughout the plan output. You may now see values redacted as (sensitive) that were previously visible, because sensitivity did not follow provider-defined sensitive attributes.
If you are transforming a value and wish to force it not to be sensitive, such as if you are transforming a value in such a way that removes the sensitive data, we recommend using the new nonsensitive function to hint Terraform that the result is not sensitive.

and
> config: New functions sensitive and nonsensitive allow module authors to explicitly override Terraform's default infererence of value sensitivity for situations where it's too conservative or not conservative enough. (#27341)

then I tried to change the local module's code at
```
$terraform-jina-jinad-aws-path/.terraform/modules/keypair/outputs.tf#L10
value = tls_private_key.generated.private_key_pem
```
to
`value = nonsensitive(tls_private_key.generated.private_key_pem)`

or
if we should output sensitive `tls_private_key.generated.private_key_pem`, there will be:
```
value = tls_private_key.generated.private_key_pem
sensitive = true
```

`terraform plan`  worked out

**Comment by alanthssss** [comment]
then I read source code of [terraform-provider-tls](https://github.com/hashicorp/terraform-provider-tls) (`mitchellh/dynamic-keys/aws` uses this provider).
it  shows that `private_key_pem` is a sensitive variable.

---

now, I should feedback to `mitchellh/dynamic-keys/aws` :Issue, PR, these things, and maybe contact the maintainer to ask them for a release.

**Comment by alanthssss** [comment]
> https://github.com/mitchellh/terraform-aws-dynamic-keys/releases/tag/v2.0.1

Here `mitchellh/terraform-aws-dynamic-keys` drafted a release with our PR half an hour ago.
I'll test the `terraform v0.15.0` now

**Comment by alanthssss** [comment]
the latest version `v2.0.1` has not be published in registry.

**Comment by alanthssss** [comment]
https://github.com/mitchellh/terraform-aws-dynamic-keys/issues/6#issuecomment-828955243

**Comment by alanthssss** [comment]
> https://github.com/hashicorp/terraform/blame/v0.15.1/CHANGELOG.md#L5-L8

This issue has been fixed in `terrafrom v0.15.1`
